ush/forecast_det.sh

```diff
@@ -40,7 +40,8 @@ FV3_GFS_det(){
   #-------------------------------------------------------
   # determine if restart IC exists to continue from a previous forecast
   RERUN="NO"
-  filecount=$(find "${RSTDIR_ATM:-/dev/null}" -type f | wc -l)
+  filecount=0
+  [[ ${CDUMP} = "gfs" ]] && filecount=$(find "${RSTDIR_ATM}" -type f | wc -l)
```

Comment:
This is reverting a fix we made to these lines.

Reply:
Please keep the version in `develop` and undo this change.
Or atleast explain why the `develop` version is inadequate for the changes related to this PR.